lib/puma/server.rb

```diff
@@ -449,10 +454,9 @@ def process_client(client)
       requests = 0
 
       begin
-        if @queue_requests &&
-          !client.eagerly_finish
+        if @queue_requests && !client.eagerly_finish
 
-          client.set_timeout(@first_data_timeout)
+          client.set_timeout @first_data_timeout
           if @reactor.add client
```

Comment:
Please please don't mix style changes into a PR like this. It's an important change and it's confusing if this is just style (I believe it is) or functional.

Reply:
Removed style change and rebased